### ISSUE TITLE
Question about Intersection Preserving

### ISSUE THREAD

**Issue by yzhang-gh** [fatPeter/mini-splatting#3]
Hi, I have tried this method and it is amazing!

However, I couldn't find the code related to the "Intersection Preserving" part. As far as I can see, there are two simplification iterations (i.e. `simp_iteration1` and `2`). Both of them make use of the `imp_score` of the gaussians (the former does downsampling and the latter cuts on a percentile). Am I missing something?

**Comment by fatPeter** [comment]
Thank you for your interest.

For simplicity, we merged Intersection Preserving into Importance-weighted Sampling, as:

```python
imp_score[accum_area_max==0]=0
```
Here, accum_area_max means the 'maximum contribution area' of the corresponding Gaussian. 

In the section of Intersection Preserving, we only preserve intersection points (i.e., Gaussians with accum_area_max!=0). In other words, we remove all Gaussians with accum_area_max==0. In this implementation, we simply set prob (imp_score) as 0, thus we can just ignore these Gaussians during sampling.
